api/blueprints/routes.py

```diff
@@ -41,41 +59,26 @@ def get_genre(genre_id):
     return jsonify(object_as_dict(genre))
 
 
-@bp.route(f'{API_BASE_PATH}/lines/<string:line_id>', methods=['GET'])
-def get_line(line_id):
-    line = Line.query.get_or_404(line_id)
-    return jsonify(object_as_dict(line))
+@bp.route(f'{API_BASE_PATH}/lines/<int:conv_id>', methods=['GET'])
```

Comment:
What do you think about using the route `/conversation/:id/lines` for this functionality? It'll avoid confusion and we could keep around `/lines/:id` to access single lines (if we ever need to)